IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -110,28 +117,10 @@ - (BOOL)saveSSOStateWithConfiguration:(MSIDConfiguration *)configuration
 
     MSID_LOG_VERBOSE(context, @"(Legacy accessor) Saving SSO state");
 
-    BOOL result = [self saveIDTokenWithConfiguration:configuration response:response context:context error:error];
-    result &= [self saveRefreshTokenWithConfiguration:configuration response:response context:context error:error];
+    BOOL result = [self saveRefreshTokenWithConfiguration:configuration response:response context:context error:error];
```

Comment:
What if error occurs on both this and the next call?
It prob doesn't matter as they are both likely to be a same error but I see first error message getting squashed.

Reply:
fixed